crates/rolldown/src/stages/generate_stage/code_splitting.rs

```diff
@@ -238,9 +237,21 @@ impl<'a> GenerateStage<'a> {
               a_should_be_first
             }
           }
-          _ => chunks[*a].exec_order.cmp(&chunks[*b].exec_order),
+          (
+            ChunkKind::EntryPoint { is_user_defined: a_is_user_defined, .. },
+            ChunkKind::EntryPoint { is_user_defined: b_is_user_defined, .. },
+          ) => {
+            if *a_is_user_defined && *b_is_user_defined {
+              // Using user specific order of entry
```

Comment:
Here fixed user specific order of entry

Reply:
Should we make sure the entry asset order is the same as the user's `input` option?